google-cloud-tools-plugin/resources/messages/CloudToolsBundle.properties

```diff
@@ -217,6 +217,7 @@ cloud.libraries.dialog.title=Add Cloud Libraries
 cloud.libraries.documentation.link=Documentation
 cloud.libraries.menu.action.text=Add Cloud libraries...
 cloud.libraries.menu.action.description=Select the Cloud libraries to download and include in your application
+cloud.libraries.menu.action.disabled.description=Maven project and Java 8 required to enable selection of the cloud libraries
```

Comment:
The Java 8 part of this message only applies if its App Engine standard. So perhaps split this message up into several cases or just make the wording more general?

@patflynn wdyt?

Reply:
see the "supported platforms" section of the [client library github page](https://github.com/GoogleCloudPlatform/google-cloud-java). I think it supports my comment. So I would go ahead and make the change.

Reply:
sounds good, I added separate messages.